libs/wingc/src/type_check/jsii_importer.rs

```diff
@@ -83,7 +88,7 @@ impl<'a> JsiiImporter<'a> {
 				PrimitiveType::Boolean => self.wing_types.bool(),
 				PrimitiveType::Any => self.wing_types.anything(),
 				PrimitiveType::Json => self.wing_types.json(),
-				PrimitiveType::Date => todo!(),
+				PrimitiveType::Date => self.wing_types.anything(), // TODO!!!!!!!!!!!!!!!!!!
```

Comment:
Without this we can't really import `aws-cdk-lib`, so I guess `anything()` is what we need right now, but we need to remember this :-) I'll open an issue.